concrete/src/File/FileServiceProvider.php

```diff
@@ -56,5 +57,16 @@ public function register()
                 ]
             );
         });
+
+        $this->app->bindIf(SanitizerOptions::class, function (Application $app) {
+            $config = $app->make('config');
+            $options = $app->build(SanitizerOptions::class);
```

Comment:
You might want to use `$this->app->extend(SanitizerOptions::class, function(SanitizerOptions $options) { ... });`

Reply:
> You might want to use `$this->app->extend(SanitizerOptions::class, function(SanitizerOptions $options) { ... });`

Why?